Fastlane/testing_lanes.rb

```diff
@@ -56,18 +61,18 @@
       output_types: '',
       # xcodebuild_formatter: '', # Add this to get verbose logging by disabling xcbeautify.
       suppress_xcode_output: false,
-      buildlog_path: ENV['BITRISE_DEPLOY_DIR'],
+      buildlog_path: ENV['BITRISE_DEPLOY_DIR'], # By configuring `BITRISE_DEPLOY_DIR` we make sure our build log is deployed and available in Bitrise.
       prelaunch_simulator: true,
-      xcargs: "-clonedSourcePackagesDirPath #{sourcePackagesDir} -parallel-testing-enabled NO -retry-tests-on-failure -test-iterations 3",
+      xcargs: "-clonedSourcePackagesDirPath #{source_packages_dir} -parallel-testing-enabled NO -retry-tests-on-failure -test-iterations 3",
       include_simulator_logs: false, # Needed for this: https://github.com/fastlane/fastlane/issues/8909
       result_bundle: true,
       output_directory: "#{ENV['PWD']}/build/reports/",
-      derived_data_path: "#{ENV['PWD']}/build/derived_data", # Set buildlog and derived data path to fix permission issues on Bitrise
-      package_path: options[:package_path],
+      derived_data_path: "#{ENV['PWD']}/build/derived_data", # Set buildlog and derived data path to fix permission issues on Bitrise.
+      package_path: options[:package_path], # Optional path to the SPM package to test.
       build_for_testing: options.fetch(:build_for_testing, nil),
       test_without_building: options.fetch(:test_without_building, nil),
       disable_package_automatic_updates: true, # Makes xcodebuild -showBuildSettings more reliable too.
-      skip_package_dependencies_resolution: options.fetch(:disable_automatic_package_resolution, false)
+      skip_package_dependencies_resolution: true
```

Comment:
Hm, I don't understand why we'd always skip resolution? Is this handled by the step above now?

Reply:
Exactly, that's now handled earlier.